lib/schemapack.js

```diff
@@ -260,9 +263,9 @@ class SchemaPack {
       specComponentsAndUtils,
       authHelper,
       schemaCache = {
-        schemaResolutionCache: this.schemaResolutionCache,
         schemaFakerCache: this.schemaFakerCache,
-        schemaValidationCache: this.schemaValidationCache
+        schemaValidationCache: this.schemaValidationCache,
+        analytics: this.analytics
```

Comment:
Should analytics be part of schemaCache object?

Reply:
It ideally shouldn't, but since schema cache was already being passed around in the code felt it would be easier to use it. 
We can call it something else and make it multi-purpose as the purpose was mostly just organising.

Reply:
Got it! Should be fine then for now.